File: query_string/friday.js
var string = "title=jaws&page=5";
var params = string.split(/\W/);
var hash = {};
var title = params[1]
var page = params[3]

hash[:title] = title;
hash[:page] = page;
// pairsString.forEach(function(pair)
// {
//    pairs.push(pair.split('='));
// });
// pairs.forEach(function(pair)
// {
//    params[pair[0]] = pair[1];
// });

File: query_string/friday.rb
require 'pry'
# "title=jaws&page=5"
# movie =
# {
#    :title => 'Jaws',
#    :page => 5,
# }

jaws = {}
params = "title=jaws&page=5"
string = params.split(/\W/)

jaws[:title] = string[1]
jaws[:page] = string[3]
puts jaws

binding.pry
